hal/src/main/native/sim/Interrupts.cpp

```diff
@@ -231,7 +235,7 @@ static int64_t WaitForInterruptDigital(HAL_InterruptHandle handle,
 
   // Cancel our callback
   SimDIOData[digitalIndex].value.CancelCallback(uid);
-  synchronousInterruptHandles->Free(dataHandle);
+  (void)synchronousInterruptHandles->Free(dataHandle);
```

Comment:
Doing this is new for us.  I’m not opposed to it, but it might be better to disable the warning globally.

Reply:
Since this is in a cpp file I can just disable that warning at the top of the file.